data/fro/Wallensköld_1891_CononBethune-Chansons2/stemma.gv

```diff
@@ -0,0 +1,39 @@
+digraph {
+
+II[color="grey"]
+1[label="", color="grey"]
+2[label="", color="grey"]
+3[label="", color="grey"]
+4[label="", color="grey"]
+5[label="", color="grey"]
+6[label="", color="grey"]
+7[label="", color="grey"]
+8[label="", color="grey"]
+9[label="", color="grey"]
+10[label="", color="grey"]
+
+
+II->1;
+1->H;
+1->y;
+II->2;
+2->3;
+3->C;
+3->4;
+4->U;
+4->I;
+2->5;
+5->O;
+5->6;
+6->7;
+7->x;
```

Comment:
Do "x" and "V" belong to one or two transmission branches?

Reply:
It is a bit ambiguous, but the editor writes:

> «Quant aux nouveaux mss. x et y, le premier est étroitement apparenté à O»

which isn't a very precise expression of relation, but can be interpreted (a bit by default) as, «they have a direct common ancestor».